### DISCUSSION TITLE
In the UI, how do i order tests alphabetically ?

### DISCUSSION THREAD

**Discussion by zeid0ne** [vitest-dev/vitest#5583] [Q&A]
Hello,

When i run my test with vitest --ui, they are all displayed in a random order.

https://private-user-images.githubusercontent.com/38461940/324242258-ffd1abe1-33e0-4f92-a34a-364a5b94e4d6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM3NzA2NjksIm5iZiI6MTcxMzc3MDM2OSwicGF0aCI6Ii8zODQ2MTk0MC8zMjQyNDIyNTgtZmZkMWFiZTEtMzNlMC00ZjkyLWEzNGEtMzY0YTViOTRlNGQ2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDIyVDA3MTkyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM2YjAwNGY4N2M5ZWQ0MTNlNDM4M2I2MTk0OWIwZTY0ZjAwZGEzNWM0YzE1MTM4ZDVlOGMzMjk2MGQ2OGIxYzYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.L9ZD9fupQtMeYIjeLLeraR7RGTu7mZIsmL1n0-qFbbo

Is it possible to order them alphabetically ?

Thanks

**Comment by sheremet-va** [comment]
It is not possible, not. You can open a feature request, or create a PR for this feature through 😄
